packages/in_app_purchase/ios/Classes/FIAPaymentQueueHandler.h

```diff
@@ -30,6 +30,7 @@ typedef void (^UpdatedDownloads)(NSArray<SKDownload *> *downloads);
 // Can throw exceptions if the transaction type is purchasing, should always used in a @try block.
 - (void)finishTransaction:(nonnull SKPaymentTransaction *)transaction;
 - (void)restoreTransactions:(nullable NSString *)applicationName;
+- (NSArray<SKPaymentTransaction *> *)getPendingTransactions;
```

Comment:
Consider changing name from "getPendingTransactions" to "getUnfinishedTransactions" or "unfinishedTransactions". Technically, transactions with `purchased` and `failed` states are not "pending" any more.

Reply:
done

Reply:
Actually, in Apple's doc https://developer.apple.com/documentation/storekit/skpaymentqueue/1506026-transactions?language=objc
It says pending transactions, should we keep this same?